homeassistant/components/geniushub/water_heater.py

```diff
@@ -79,9 +84,7 @@ def name(self):
     def device_state_attributes(self):
         """Return the device state attributes."""
         tmp = self._boiler.__dict__.items()
-        state = {k: v for k, v in tmp if k in GH_DEVICE_STATE_ATTRS}
-
-        return {'status': state}
+        return {'status': {k: v for k, v in tmp if k in GH_STATE_ATTRS}}
 
```

Comment:
We should add  `should_poll` set to `False` here, right?

Reply:
Woops  - good spot!